tests/tests/blocks.rs

```diff
@@ -41,23 +29,15 @@ use std::ops::Deref;
 
 #[tokio::test]
 async fn block() {
-    // setup test data in the node
-    let block = CompressedBlock::default();
-    let id = block.id();
-    let mut db = Database::default();
-    db.storage::<FuelBlocks>().insert(&id, &block).unwrap();
-    db.storage::<SealedBlockConsensus>()
-        .insert(&id, &Consensus::PoA(Default::default()))
-        .unwrap();
-
     // setup server & client
-    let srv = FuelService::from_database(db, Config::local_node())
+    let srv = FuelService::from_database(Database::default(), Config::local_node())
         .await
         .unwrap();
     let client = FuelClient::from(srv.bound_address);
 
     // run test
-    let id_bytes: Bytes32 = id.into();
+    let block = client.block_by_height(0).await.unwrap().unwrap();
```

Comment:
isn't this now basically the same as the next test "`get_genesis_block`"?

Reply:
Partially, it checks the `block` endpoint, which can receive the block while the `get_genesis_block` checks that we can get the genesis, but you are right, they are almost the same. Updated them to show the difference more.